neurots/preprocess/validity_checkers.py

```diff
@@ -39,6 +43,12 @@ def check_num_seg(params, distrs):
         )
 
 
+@register_global_validator()
+def validate(params, distrs):
```

Comment:
```suggestion
def check_input_validity(params, distrs):
```
Just to be consistent with our own convention.